README.md

```diff
@@ -1,6 +1,11 @@
 [![CircleCI](https://circleci.com/gh/bartholomews/scalatestudo/tree/master.svg?style=svg)](https://circleci.com/gh/bartholomews/scala-iso/tree/master)
 [![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://github.com/bartholomews/spotify4s/blob/master/LICENSE)
 
+# Disclaimer
+
+Dirty fork from: https://github.com/bartholomews/scala-iso
+Sole purpose is to gain a scala3 release since the upstream project seems abandoned.
+
```

Comment:
I guess this is not relevant if it's going back here ?

Reply:
:+1: